core/src/main/java/com/sequenceiq/cloudbreak/job/existingstackfix/ExistingStackPatcherJobInitializer.java

```diff
@@ -0,0 +1,32 @@
+package com.sequenceiq.cloudbreak.job.existingstackfix;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.job.AbstractStackJobInitializer;
+
+@Component
+public class ExistingStackPatcherJobInitializer extends AbstractStackJobInitializer {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ExistingStackPatcherJobInitializer.class);
+
+    @Inject
+    private ExistingStackPatcherConfig config;
+
+    @Inject
+    private ExistingStackPatcherJobService jobService;
+
+    @Override
+    public void initJobs() {
+        if (config.isExistingStackPatcherEnabled()) {
+            LOGGER.info("Scheduling stack patcher jobs");
+            getAliveAndNotDeleteInProgressStacksStream()
```

Comment:
I see a logic where we are skipping the StackPatch operation if it was already applied, but we still schedule jobs for them until the feature is disabled?